packages/date-picker/test/keyboard-input.test.js

```diff
@@ -461,37 +462,95 @@ describe('keyboard', () => {
       expect(result.getFullYear()).to.equal(99);
     });
 
-    it('should parse three digits with short year', async () => {
-      await sendKeys({ type: '6/20/99' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(1999);
-    });
-
-    it('should parse three digits with short year 2', async () => {
-      await sendKeys({ type: '6/20/20' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(2020);
-    });
-
-    it('should parse three digits with short year 3', async () => {
-      await sendKeys({ type: '6/20/1' });
-      const result = focusedDate();
-      expect(result.getFullYear()).to.equal(2001);
-    });
-
     it('should parse three digits with negative year', async () => {
       await sendKeys({ type: '6/20/-1' });
       const result = focusedDate();
       expect(result.getFullYear()).to.equal(-1);
     });
 
     it('should parse in base 10', async () => {
+      datepicker.i18n = {
+        ...datepicker.i18n,
+        referenceDate: extractDateParts(new Date(2022, 0, 1)),
+      };
       await sendKeys({ type: '09/09/09' });
       const result = focusedDate();
       expect(result.getFullYear()).to.equal(2009);
       expect(result.getMonth()).to.equal(8);
       expect(result.getDate()).to.equal(9);
     });
+
+    it('should throw when passing a year < 0', () => {
+      expect(() => getAdjustedYear(today, -1, 1, 1)).to.throw(Error);
+    });
+
+    it('should throw when passing a year >= 100', () => {
+      expect(() => getAdjustedYear(today, 100, 1, 1)).to.throw(Error);
```

Comment:
Let's also add tests for calling ` getAdjustedYear` without `month` and `day`, or with `month` only.
If we add fallback values as I suggested in the above comment, then it should work as expected.

Reply:
Added tests with the latest commit